src/components/__test__/DeleteMotor.test.js

```diff
@@ -0,0 +1,64 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import React from 'react';
+import {
+  render, fireEvent, waitFor, screen,
+} from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import motorReducer from '../../redux/slice/motorSlice';
+import DeleteMotor from '../DeleteMotor/DeleteMotor';
+
+jest.mock('swiper/react', () => ({
+  // eslint-disable-next-line react/prop-types
+  SwiperSlide: ({ children }) => <div>{children}</div>,
+}));
+test('DeleteMotor component deletes a motor item', async () => {
+  const initialState = {
+    motor: {
+      motorData: [
+        {
+          id: 1,
+          brand_name: 'Test Brand 1',
+          model_no: 'Model 1',
```

Comment:
Please implement Jest snapshot test.
Jest snapshot testing is a feature provided by the Jest testing framework that allows you to capture and compare snapshots of the rendered output of your React components. This technique helps you detect unintended changes in the UI over time and makes it easier to identify regressions in your application's visual appearance. Its user interface test. for example`test('YourComponent snapshot test', () => {
  const { asFragment } = render(<YourComponent />);
  expect(asFragment()).toMatchSnapshot();
});
`